### PR TITLE
Export base image tag info interface

### PR DESCRIPTION
**Pull request by roman-mazur**
Change-type: patch
Signed-off-by: Roman Mazur <roman@balena.io>

### CODE REVIEW

**Comment by roman-mazur** [comment]
Missed this in #32.

File: src/index.ts [hunk 1]
```diff
@@ -15,7 +15,7 @@
  * limitations under the License.
  */
 import Builder from './builder';
-import { BuildHooks } from './plugin';
+import { BuildHooks, FromTagInfo } from './plugin';
```
Comment (Page-) [inline src/index.ts]:
```suggestion
export { BuildHooks, FromTagInfo } from './plugin';
```

Comment (roman-mazur) [reply src/index.ts]: Adjusted

**Comment by roman-mazur** [comment]
@balena-ci retest please

**Pull request merged**